public/xash3d_mathlib.h

```diff
@@ -80,8 +80,9 @@ GNU General Public License for more details.
 #define Q_round( x, y )	(floor( x / y + 0.5f ) * y )
 #define Q_rint(x)		((x) < 0.0f ? ((int)((x)-0.5f)) : ((int)((x)+0.5f)))
 
-#ifdef XASH_IRIX
+#if defined( XASH_IRIX ) && defined( __GNUC__ )
```

Comment:
Same as above. You can just use simple and readable `#if XASH_IRIX && __GNUC__`.